CHANGELOG.md

```diff
@@ -46,7 +47,6 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Pydantic validation of image paths provided in case load config file
 - Info on the user which created a ClinVar submission, when available
 - Associate .d4 files to case individuals when loading a case via config file
-- Option update path to .d4 files path for individuals of an existing case using the commands line
```

Comment:
This should be left as it is now, I think?

Reply:
Yes, the PR lived on rebase for too long?!